### PR TITLE
formatted table properly. added cross check marks

### PR DESCRIPTION
**Pull request by RAJU2529**
this is my own PR,
01.
 I added   Checkmarks only for Business  edition  if  under Professional and Enterprise are already added Checkmarks.

02.
 I added   Crossmarks only for Business  edition  if   under Professional and Enterprise are already added Crossmarks.

03.
Removed the following words
 **Only for mobile application management (MAM)**
**Provisioning only** 

04.
Added footnotes  
**A- Only for mobile application management (MAM)**
**B- Provisioning only**

### CODE REVIEW

File: windows/client-management/mdm/configuration-service-provider-reference.md [hunk 1]
```diff
@@ -2627,6 +2623,8 @@ The following list shows the CSPs supported in HoloLens devices:
 <hr>
 
  Footnotes:
+- A - Only for mobile application management (MAM)		
+- B - Provisioning only
```
Comment (illfated) [inline windows/client-management/mdm/configuration-service-provider-reference.md:2627]: This PR looks very good to me format-wise, we only need to remove some redundant whitespace (blanks) here:
```suggestion
- A - Only for mobile application management (MAM)
- B - Provisioning only
```

**Review by illfated — COMMENTED** [review]
Seems like a very good idea to implement the footnotes A and B, to follow the codestyle of numbering the Windows versions.

Rich Diff view, to simplify the review process:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9632/files?short_path=b0c1433#diff-b0c143366f62d1fa18dac35243db9180ef9c6dbcf5b896a4f8147d91ce4f3112

Knowing that I don't have any working experience with CSP and provisioning, I select to merit the changes based on the formatting, rather than content. I defer to the document author or manager for content approval.

**Comment by illfated** [comment]
( Ref. our conversations in issue ticket **non-functioning links** #9584 )

**Comment by theonlycoder** [comment]
Thank you for taking the time to put this together following on from my original issue. 

Before merging, has anyone been able to confirm for sure whether my assumption that all CSPs that are  supported in Pro edition would also be supported in Business edition is correct?

**Comment by illfated** [comment]
Thank you for asking, that is appropriate and useful. Count me out, because I am not fluent in this topic, as well as not having used this technology myself. All I can provide is formatting advice, so I defer to the MS Docs team members, the document author or manager, or possibly the tech writers for MS Docs.

**Comment by illfated** [comment]
( also ref. issue ticket **Confusion regarding AppLocker CSP support with Windows 10 Business edition** #9560 )

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/client-management/mdm/configuration-service-provider-reference.md [hunk 2]
```diff
@@ -2627,6 +2623,8 @@ The following list shows the CSPs supported in HoloLens devices:
 <hr>
 
  Footnotes:
+- A - Only for mobile application management (MAM)
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/configuration-service-provider-reference.md:2626]:
```suggestion
- A - Only for mobile application management (MAM).
```

File: windows/client-management/mdm/configuration-service-provider-reference.md [hunk 3]
```diff
@@ -2627,6 +2623,8 @@ The following list shows the CSPs supported in HoloLens devices:
...
+- B - Provisioning only
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/configuration-service-provider-reference.md:2627]:
```suggestion
- B - Provisioning only.
```

File: windows/client-management/mdm/configuration-service-provider-reference.md [hunk 4]
```diff
@@ -2636,4 +2634,4 @@ The following list shows the CSPs supported in HoloLens devices:
 - 7 - Added in Windows 10, version 1909.
 - 8 - Added in Windows 10, version 2004.
 - 9 - Added in Windows 10 Team 2020 Update
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/configuration-service-provider-reference.md:2636]:
```suggestion
- 9 - Added in Windows 10 Team 2020 Update
```
```suggestion
- 9 - Added in Windows 10 Team 2020 Update.
```

File: windows/client-management/mdm/configuration-service-provider-reference.md [hunk 5]
```diff
@@ -2636,4 +2634,4 @@ The following list shows the CSPs supported in HoloLens devices:
...
-- 10 - Added in [Windows Holographic, version 20H2](/hololens/hololens-release-notes#windows-holographic-version-20h2)
\ No newline at end of file
+- 10 - Added in [Windows Holographic, version 20H2](/hololens/hololens-release-notes#windows-holographic-version-20h2)
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/configuration-service-provider-reference.md:2637]:
```suggestion
- 10 - Added in [Windows Holographic, version 20H2](/hololens/hololens-release-notes#windows-holographic-version-20h2).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@RAJU2529 - Could you please just add the periods (.) at the end of the lines? Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me, format-wise. (https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9632/files/818a12067925afaadc3bc520df2a63a3c25d6ff1?short_path=b0c1433#diff-b0c143366f62d1fa18dac35243db9180ef9c6dbcf5b896a4f8147d91ce4f3112)

I refer to the document author or manager for content change validation.

**Comment by RAJU2529** [comment]
sincere thanks to @JohanFreelancer9 for suggestions to improve this article  and Thanks to @Dansimp and @ghost.

**Pull request merged**